Sasquatch/Sasquatch.xcodeproj/project.pbxproj

```diff
@@ -1399,6 +2201,135 @@
 			};
 			name = Release;
 		};
+		B20AE3A921027D6200A1A195 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ASSETCATALOG_COMPILER_APPICON_NAME = PuppetIcon;
+				CODE_SIGN_ENTITLEMENTS = ../TestApps/Sasquatch/SasquatchObjC/SasquatchObjC.entitlements;
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				CODE_SIGN_STYLE = Automatic;
+				DEFINES_MODULE = NO;
+				DEVELOPMENT_TEAM = "$(inherited)";
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(SRCROOT)/../Vendor/iOS/**",
+					"$(SRCROOT)/../CrashLib/CrashLib",
+				);
+				GCC_PREPROCESSOR_DEFINITIONS = (
+					"DEBUG=1",
+					"$(inherited)",
+					"GCC_PREPROCESSOR_MACRO_PUPPET=1",
```

Comment:
Is this a naming convention, `GCC_PREPROCESSOR_MACRO`?

Reply:
I can't use a preprocessor macro for swift code because swift doesn't have a preprocessor. And the new `SWIFT_ACTIVE_COMPILATION_CONDITION` is swift only. The option would be to have both flags named `PUPPET`, have `IS_PUPPET` (like in earlier versions of this PR) or be more explicit and I wanted the flags to be as explicit as possible without writing documentation somewhere.

Reply:
My questions wasn't on GCC preprocessor macro. Just the name of the macro. 😄